app/_locales/en/messages.json

```
@@ -2352,6 +2352,9 @@
   "mmiBuiltAroundTheWorld": {
     "message": "MetaMask Institutional is designed and built around the world."
   },
+  "mmiNewNFTDetectedMessage": {
+    "message": "Allow MetaMask Institutional to automatically detect NFTs from Opensea and display in your wallet."
```

Looking really good! Just one question do you think it should be `OpenSea` ?
```suggestion
    "message": "Allow MetaMask Institutional to automatically detect NFTs from OpenSea and display in your wallet."
```